<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>779</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>911</width>
      <height>761</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>false</underline>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>891</width>
        <height>461</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>640</y>
        <width>891</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="clear">
         <property name="styleSheet">
          <string notr="true">QPushButton {
                              background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 1, fx: 0.5, fy: 0.5, stop: 0 rgba(116, 116, 116), stop: 1 rgba(50, 50, 50));
                              border: 15px;
                              color: rgb(255, 255, 255);
                              font: 20pt &quot;Segoe MDL2 Assets&quot;;
                              border-radius: 10px;
                          }
                          QPushButton:pressed {
                              background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 1, fx: 0.5, fy: 0.5, stop: 0 rgba(50, 50, 50), stop: 1 rgba(116, 116, 116));
                          }</string>
         </property>
         <property name="text">
          <string>Clear Text</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_2">
         <property name="styleSheet">
          <string notr="true">QPushButton {
                              background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 1, fx: 0.5, fy: 0.5, stop: 0 rgba(116, 116, 116), stop: 1 rgba(50, 50, 50));
                              border: 15px;
                              color: rgb(255, 255, 255);
                              font: 20pt &quot;Segoe MDL2 Assets&quot;;
                              border-radius: 10px;
                          }
                          QPushButton:pressed {
                              background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 1, fx: 0.5, fy: 0.5, stop: 0 rgba(50, 50, 50), stop: 1 rgba(116, 116, 116));
                          }</string>
         </property>
         <property name="text">
          <string>Clear Character</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startcam">
         <property name="styleSheet">
          <string notr="true">QPushButton {
                              background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 1, fx: 0.5, fy: 0.5, stop: 0 rgba(116, 116, 116), stop: 1 rgba(50, 50, 50));
                              border: 15px;
                              color: rgb(255, 255, 255);
                              font: 20pt &quot;Segoe MDL2 Assets&quot;;
                              border-radius: 10px;
                          }
                          QPushButton:pressed {
                              background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 1, fx: 0.5, fy: 0.5, stop: 0 rgba(50, 50, 50), stop: 1 rgba(116, 116, 116));
                          }</string>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pausecam">
         <property name="styleSheet">
          <string notr="true">QPushButton {
                              background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 1, fx: 0.5, fy: 0.5, stop: 0 rgba(116, 116, 116), stop: 1 rgba(50, 50, 50));
                              border: 15px;
                              color: rgb(255, 255, 255);
                              font: 20pt &quot;Segoe MDL2 Assets&quot;;
                              border-radius: 10px;
                          }
                          QPushButton:pressed {
                              background-color: qradialgradient(cx: 0.5, cy: 0.5, radius: 1, fx: 0.5, fy: 0.5, stop: 0 rgba(50, 50, 50), stop: 1 rgba(116, 116, 116));
                          }</string>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="text">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>480</y>
        <width>891</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPushButton" name="record_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>600</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Record</string>
      </property>
     </widget>
     <widget class="QLabel" name="img_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>421</width>
        <height>441</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Image</string>
      </property>
     </widget>
     <widget class="QLabel" name="text_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>891</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Text</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stop_record_button">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>600</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Record</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stop_video">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>600</y>
        <width>151</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop Video</string>
      </property>
     </widget>
     <widget class="QPushButton" name="play_video">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>600</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Play Video</string>
      </property>
     </widget>
     <widget class="QLabel" name="img_label_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>441</width>
        <height>441</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>Image</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
